### ISSUE TITLE
Installation failure

### ISSUE THREAD

**Issue by bxhbpdxhzGithub** [Grokzen/redis-py-cluster#257]
Downloading/unpacking redis-py-cluster
  Running setup.py egg_info for package redis-py-cluster
    warning: no files found matching 'CHANGES'
Requirement already satisfied (use --upgrade to upgrade): redis>=2.10.2 in /usr/lib/python2.6/site-packages (from redis-py-cluster)
Installing collected packages: redis-py-cluster
  Running setup.py install for redis-py-cluster
    SyntaxError: ('invalid syntax', ('/usr/lib/python2.6/site-packages/rediscluster/nodemanager.py', 151, 37, '        connection_kwargs = {k: v for k, v in self.connection_kwargs.items() if k in set(allowed_keys) - set(disabled_keys)}\n'))
    
    SyntaxError: ('invalid syntax', ('/usr/lib/python2.6/site-packages/rediscluster/client.py', 219, 83, "        servers = list({'{0}:{1}'.format(nativestr(info['host']), info['port']) for info in self.connection_pool.nodes.startup_nodes})\n"))
    
    warning: no files found matching 'CHANGES'
Successfully installed redis-py-cluster
Cleaning up...

**Comment by Grokzen** [comment]
Please read the docs regarding supported python versions for this lib https://redis-py-cluster.readthedocs.io/en/master/#supported-python-versions

TLDR: Python 2.6 is and will not be supported by this lib.
